### DISCUSSION TITLE
Show the last 10 days history for the next Saving Session period

### DISCUSSION THREAD

**Discussion by deltamelter** [BottlecapDave/HomeAssistant-OctopusEnergy#531] [Ideas]
Since the new saving sessions are looking back over 10 days to decide "normal" usage so that they know how much we "really" save,  could you add an entity to show the history for the upcoming period over the previous 10 days?  

> We’ll look at your historical smart meter data leading up to a Session, and calculate what you usually use at that time of day, on average.
If the Session is on a weekday, we’ll look at your half-hourly usage over the last 10 weekdays (excluding Saving Session days). If the Session is on a weekend, we’ll look at your half hourly usage over the last 4 weekend days.
We then use these half-hourly averages as a benchmark – so you’ll earn points for every kilowatt hour you save compared to that number during your Session.

**Comment by BottlecapDave** [comment]
Good idea. I'll convert this to a proper issue. I'm. It not sure when I'll get to it though.
